$OpenBSD$

Only provide /usr/include in the default C header search path.

--- tools/clang/lib/Frontend/InitHeaderSearch.cpp.orig	Wed Aug  1 23:49:24 2012
+++ tools/clang/lib/Frontend/InitHeaderSearch.cpp	Wed Aug  1 23:49:57 2012
@@ -210,6 +210,7 @@ void InitHeaderSearch::AddDefaultCIncludePaths(const l
     switch (os) {
     case llvm::Triple::FreeBSD:
     case llvm::Triple::NetBSD:
+    case llvm::Triple::OpenBSD:
       break;
     default:
       // FIXME: temporary hack: hard-coded paths.
